compiler/testData/ir/irText/expressions/callableReferences/suspendConversion.fir.txt

```diff
@@ -108,8 +108,6 @@ FILE fqName:<root> fileName:/suspendConversion.kt
     BLOCK_BODY
       CALL 'public final fun useSuspend (fn: kotlin.coroutines.SuspendFunction0<kotlin.Unit>): kotlin.Unit declared in <root>' type=kotlin.Unit origin=null
         fn: BLOCK type=kotlin.coroutines.SuspendFunction0<kotlin.Unit> origin=ADAPTED_FUNCTION_REFERENCE
-          VAR IR_TEMPORARY_VARIABLE name:tmp_0 type:<root>.C [val]
-            CONSTRUCTOR_CALL 'public constructor <init> () [primary] declared in <root>.C' type=<root>.C origin=null
```

Comment:
Funny thing about these FIR tests: in the FUNCTION_REFERENCE below, the constructor call used to be copied instead of reading this variable. I have no idea how this is possible since the old code clearly assigns an IrGetValueImpl to extensionReceiver.